data/json/furniture.json

```diff
@@ -5878,7 +5910,7 @@
     "id": "f_server",
     "looks_like": "f_utility_shelf",
     "name": "server stack",
-    "description": "This is a big pile of computers.  They're all turned off.",
+    "description": "This is a big pile of computers.  They're all turned off...forever.",
```

Comment:
```suggestion
    "description": "This is a big pile of computers.  They're all turned off... forever.",
```

Reply:
For the 'They're all turned off...forever' - well, that's a style thing I've always used. Is there a common style choice across CDDA to have blah...[space]blah?

Reply:
Punctuation marks are separated from the following words.

Reply:
I'd about the whole "... forever" thing anyway; I don't think it adds anything to the description, and as powered furnishings are added it may become outdated.